<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="VR" failures="2" tests="2" name="Test(failed)" time="42.738" errors="0" timestamp="2024-11-30T17:43:15 IST">
  <testcase classname="testClass.TC_003_LoginDDT" name="verifyloginDDT" time="18.419">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testClass.TC_003_LoginDDT.verifyloginDDT(TC_003_LoginDDT.java:66)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verifyloginDDT -->
  <testcase classname="testClass.TC_003_LoginDDT" name="verifyloginDDT" time="18.409">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testClass.TC_003_LoginDDT.verifyloginDDT(TC_003_LoginDDT.java:66)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verifyloginDDT -->
</testsuite> <!-- Test(failed) -->
